privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -882,19 +884,23 @@ def update(self, param, reset_failcount=True):
                 self.add_tokeninfo(WEBAUTHNINFO.ATTESTATION_SERIAL,
                                    attestation_cert.get_serial_number())
 
-                if not description:
-                    cn = web_authn_credential.attestation_cert.subject.get_attributes_for_oid(x509.NameOID.COMMON_NAME)
-                    description = cn[0].value if len(cn) else None
+                cn = web_authn_credential.attestation_cert.subject.get_attributes_for_oid(x509.NameOID.COMMON_NAME)
+                automatic_description = cn[0].value if len(cn) else None
 
-            self.set_description(description or DEFAULT_DESCRIPTION)
+            # If no description has already been set, set the automatic description or the
+            # description given in the 2nd request
+            if not self.token.description:
+                self.set_description(getParam(param, "description", default=automatic_description))
 
             # Delete all challenges. We are still in enrollment, so there
             # *should* be only one, but it can't hurt to be thorough here.
             for challengeobject in challengeobject_list:
                 challengeobject.delete()
             self.challenge_janitor()
+            # Reset clientwait rollout_state
+            self.token.rollout_state = ""
         else:
-            self.set_description("WebAuthn initialization")
+            raise ParameterError("regdata and or clientdata provided but token not in clientwait rollout_state.")
```

Comment:
like above please add the token serial

Reply:
It could also be that one of `regdata` or `clientdata` is missing.

Reply:
will do

Reply:
see above. Since it is an exception, I do not add this.

The text tries to reflect this by useind "and or".